examples/community.html

```diff
@@ -9,6 +9,11 @@
   <!-- /build -->
   <link href="common/css/style.css" rel="stylesheet">
   <title>Community Examples</title>
+  <script>
+    // examples don't load on https
+    if (location.protocol == 'https:')
```

Comment:
that's an interesting approach. I for one would be surprised why this page suddenly switched and messes with my browser history. What do you think about a banner that explains the situation, asks for a click, and is only visible on `https` ?